test/project_types/script/forms/ask_script_uuid_test.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Forms::AskScriptUuid do
+  include TestHelpers::FakeFS
+  describe ".ask" do
+    let(:context) { TestHelpers::FakeContext.new(root: Dir.mktmpdir) }
+    let(:existing_scripts) { [] }
+
+    subject do
+      result = nil
+      capture_io { result = Script::Forms::AskScriptUuid.ask(context, existing_scripts, {}) }
+      result
+    end
+
+    describe("when asking script connection") do
+      describe("when number of scripts == 0") do
+        it("should not prompt the user to confirm if they want to replace existing scripts") do
+          CLI::UI::Prompt.expects(:confirm).never
+          CLI::UI::Prompt.expects(:ask).never
+          subject
+        end
+      end
+
+      describe("when number of scripts > 0") do
```

Comment:
Might be better worded as "when there is 1 script"?

Reply:
Do you mean "When there is at least 1 script"? I see that the test case only has a single script  in the "existing scripts" variable, but this test case also covers the situation where an app has multiple scripts, i.e.:

![](https://screenshot.click/srcgithub.comShopifyshopify-clibinshopify_script_push_2021-11-24_15-59-33.png)

Reply:
I meant _when there is 1 script_ so the describe block describes the actual test that is running, not the range of cases the test intends to cover.